fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/mixin/resource/loader/MixinPackScreen.java

```diff
@@ -28,7 +28,7 @@
 import net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator;
 
 @Mixin(PackScreen.class)
-public class MixinAbstractPackScreen {
+public class MixinPackScreen {
 	@Inject(method = "method_29672", at = @At("HEAD"), cancellable = true)
 	private void addPackEntry(PackListWidget packListWidget, ResourcePackOrganizer.Pack pack, CallbackInfo info) {
 		if (pack.getSource() == ModResourcePackCreator.RESOURCE_PACK_SOURCE) {
```

Comment:
Hmm, I wonder how this would hurt as you cannot order scanned mod packs

Reply:
What do you mean?

The inability to order mod resource packs was already a thing before this PR.